### ISSUE TITLE
steam client downloading something after 3/21/2018 client update

### ISSUE THREAD

**Issue by lynx29** [ValveSoftware/steam-for-linux#5417]
#### Your system information

* Steam client version (build number or date): march 22 2018 
* Distribution (e.g. Ubuntu): SolusOS
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
After the march 21 2018 update I got an issue with stem client eating all my bandwidth checking with ss command 

users:(("gameoverlayui",pid=2763,fd=93),("steam",pid=2092,fd=93))
ESTAB      173808 0         192.168.1.14:37667            220.243.200.109:http           
                                          
#### Steps for reproducing this issue:

1. 
2. 
3.

**Comment by h1z1** [comment]
Lol you assigned this to _triage_ ???  (aka blackhole/go-away/lolkthnxbye)

```
inetnum:        220.243.0.0 - 220.243.255.255
netname:        CHINANETCENTER
descr:          ChinaNetCenter Ltd.
admin-c:        ZM1723-AP
tech-c:         ZM1723-AP
country:        CN
mnt-by:         MAINT-CNNIC-AP
mnt-irt:        IRT-CNNIC-CN
status:         ALLOCATED PORTABLE
last-modified:  2016-11-01T06:12:03Z
source:         APNIC

```

**Comment by lynx29** [comment]
Yup i know it's an http connection but why the steam process  connect to that ip address and it keeps downloading data. The steam interface that only open is the library nothing else. It keeps downloading while i was playing dota 2. This only happens after steam client update of 3/21/2018. BTW it's not only that ip address that it  would connect to sometimes it would connect to highwind cdn and download data even if i was playing game.

**Comment by lynx29** [comment]
I think its not the steam client that trigger the connection for the download. This morning i run wireshark and let steam run with out opening any game steam only on library interface keeping it on this state for 2 hours no download happening. but after opening dota 2 download happens triggering connection to port 80 even when already playing a game but checking with ss command the download falls on steam process. Should i report this to dota 2 or here steam-for-linux.

**Comment by h1z1** [comment]
Depends, if you ran wireshark what was the URL?

**Comment by AlejandroC90** [comment]
This issue still persists, 40 minutes in game, in a heavy teamfight, the client starts to download and gives lag, had to restart client

**Comment by h1z1** [comment]
It's unlikely Valve will respond to this given the climate.  I'm not in China yet simply resolving those addresses caused probing from a few hosts in China.  You could try adding a rule to your firewall, rejecting any traffic to that ip.
